package-lock.json

```diff
@@ -5664,7 +5664,8 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
I'm not sure this is a good idea - they may be based on an older version of NPM and only want to upgrade in their own time (and in a separate PR). It feels like we're conflating concerns to an extent.

Reply:
Okay that's a fair point. Hopefully this'll be addressed as part of their beta update process.